src/main/java/ch/njol/skript/expressions/ExprDrops.java

```diff
@@ -51,24 +52,33 @@
  * @author Peter Güttinger
  */
 @Name("Drops")
-@Description("Only works in death events. Holds the drops of the dying creature. Drops can be prevented by removing them with \"remove ... from drops\", e.g. \"remove all pickaxes from the drops\", or \"clear drops\" if you don't want any drops at all.")
+@Description("Only works in death events (Also block break events in 1.12+). Holds the drops of the dying creature. Drops can be prevented by removing them with \"remove ... from drops\", e.g. \"remove all pickaxes from the drops\", or \"clear drops\" if you don't want any drops at all.")
 @Examples({"clear drops",
-		"remove 4 planks from the drops"})
-@Since("1.0")
-@Events("death")
+		"remove 4 planks from the drops", "on break of diamond ore:", "\tclear drops"})
+@Since("1.0, INSERT VERSION (break event drops)")
+@Events("break(1.12+)/death")
```

Comment:
Events is an array, and needs raw event names. You can ignore the fact that MC 1.12 or newer here.

Reply:
Good to know. I looked up that event, and its registered as "break / mine" so I hope that was the correct one that needs to go in there